docs/docs/building-services/gramine.md

```diff
@@ -74,21 +98,32 @@ You can specify the files' content in the MarbleRun manifest:
 ...
 ```
 
-Note that Gramine also allows to store files encrypted on the host's file system. The so called `protected files` require to initialize the protected files key by writing it hex-encoded to the virtual `protected_files_key` device:
+Note that Gramine also allows to store files encrypted on the host's file system. The so encrypted filesystem requires to initialize an encrypted files key by writing it to the virtual `protected_files_key` device.
+For debugging purposes, you can define the key directly in the Gramine manifest.
 
+```toml
+fs.insecure__keys.[KEY_NAME]] = "[16-byte hex value]"
+fs.mounts = [
+  # ...
+  { type = "encrypted", path = "/secrets", uri = "file:/path/to/local/directory", key_name = "[KEY_NAME] },
+  # ...
+]
 ```
-sgx.protected_files_key = "[16-byte hex value]"
-sgx.protected_files.[identifier] = "[URI]"
-```
-You can see how this key can be initialized with MarbleRun in the [nginx example](https://github.com/edgelesssys/marblerun/tree/master/samples/gramine-nginx).
+
+MarbleRun allows provisioning the encrypted files key at runtime to your enclave.
```

Comment:
I would maybe highlight it a bit more that this is actually the recommended way by Gramine and `insecure__keys` is just here for a quick-start and minimal sample.